### PR TITLE
A handful of fixes to address additional reported crashes

### PR DESCRIPTION
**Pull request by rictic**
Should fix the issues reported in by @yonathan9669 in #13

### CODE REVIEW

File: test/fixtures/simple-element.html.out [hunk 1]
```diff
@@ -11,19 +11,25 @@
...
+      <div><span>{{intAttr}}</span> times I tell you <span>{{strAttr}}</span></div>
+      <div>Yes <span>{{intAttr}}</span> times I tell you <span>{{strAttr}}</span></div>
+      <div>Yes <span>{{intAttr}}</span> times I tell you <span>{{strAttr}}</span>.</div>
     </template>
   </template>
 </dom-module>
 <script>
   // this is a comment
-  Polymer({
+  Polymer(//TODO(polyup): Need to add an observer of z.v.k, calling somethingsUp.
```
Comment (justinfagnani) [inline test/fixtures/simple-element.html.out]: can these comments go directly above the Polymer() call?

**Comment by justinfagnani** [comment]
LGTM

File: test/fixtures/simple-element.html.out (hunk 1 above)
Comment (rictic) [inline test/fixtures/simple-element.html.out]: +1. Done

**Comment by rictic** [comment]
Thanks for the review!

**Pull request merged**